Changed conditional to check for the word 'From'

AOrcilla opened the pull request:
using == instead of  .startswith()

AOrcilla committed:
Changed conditional to check for the word 'From' using == instead of .startswith()

dtil-gh commented on dict-keys.py:
```diff
@@ -9,8 +9,8 @@
-        if wrd.startswith("From:"): #Reads through split words to identify if the line starts with 'From:'
-            wrd = lin.strip()
+        if wrd == "From": #Reads through split words to identify if the line starts with 'From:'
+            wrd = lin.strip() #strips whitespace
```
Delete this line, I believe it's unnecessary to strip spaces within a list that has already been stripped from line 9.

AOrcilla replied:
Makes sense, deleting line 13.

AOrcilla committed:
Deleted line 13

wrd = lin.strip() was already called previously so there was no need to have it in this conditional statement.

AOrcilla reviewed (commented):
Needs approval. @dtil-gh

dtil-gh commented:
looks sexy
